app/views/deploys/recent.html.erb

```diff
@@ -25,6 +26,15 @@
           <option value="">All</option>
         </select>
       </div>
+      <div class="timeline-filter col-sm-4" >
+        <label for="tlf-status">TIME:</label>
+        <% User::TIME_FORMATS.each do |time_fmt| %>
```

Comment:
I'd recommend spelling out short works, fmt -> format ... seems to be a ruby standard